commit a2f4f7fb9693a0c61593ccc698a72d37695c7e0d
Author: Richard W.M. Jones <rjones redhat com>
Date:   Fri Oct 19 10:31:52 2012 +0100

    windows: Fix creation of /Temp/V2V directory
    
    case_sensitive_path was not defined when called on a path where the
    final element doesn't exist.  In libguestfs >= 1.16.29 it was changed
    to return non-NULL if the final element doesn't exist so that creation
    of new files works.
    
    Fixes RHBZ#868073
    (cherry picked from commit 06f8b55e8d9e159f10beeeb6cbd51facd92eb9b5)

diff --git a/lib/Sys/VirtConvert/Converter/Windows.pm b/lib/Sys/VirtConvert/Converter/Windows.pm
index ce89a67..d4885be 100644
--- a/lib/Sys/VirtConvert/Converter/Windows.pm
+++ b/lib/Sys/VirtConvert/Converter/Windows.pm
@@ -470,12 +470,8 @@ sub _upload_files
     foreach my $d ('Temp', 'V2V') {
         $path .= '/'.$d;
 
-        eval { $path = $g->case_sensitive_path($path) };
-
-        # case_sensitive_path will fail if the path doesn't exist
-        if ($@) {
-            $g->mkdir($path);
-        }
+        $path = $g->case_sensitive_path($path);
+        $g->mkdir_p($path);
     }
 
     foreach my $file (@fb_files) {
